contribute.md

```diff
@@ -1,10 +1,110 @@
----
-title: Contribute
-layout: default
----
+## Contributing to the Rackspace Design System repository
```

Comment:
We might want to leave the metadata at the top of this to render for the site. Not sure if folks want it to live on the site for now or live in the repo where it will live in the end product.

Reply:
I can't imagine a contribute.md living on the site... although now I think about it, that might be a good idea seeing as the whole project is all about contributing. @araiford ?

Reply:
@asettle, there's a Getting Started: Get Involved section on the final site layout, and this page was filling in for that on the current site. Just not sure if this page should be on the site in the meantime, really.